requirements/mypy.in

```diff
@@ -31,3 +31,6 @@ types-stripe
 types-zxcvbn
 
 importlib-metadata ; python_version < "3.10"  # for SQLAlchemy
+
+https://github.com/typeddjango/django-stubs/archive/c0414c7b025b77caf9f72dbaf9e8471e11aad085.zip#egg=django-stubs-ext&subdirectory=django_stubs_ext
+https://github.com/typeddjango/django-stubs/archive/18a055124a2212e13c97bc1a83cc36b4e8104a97.zip#egg=django-stub==1.12.0+git
```

Comment:
Presumably these should use the same commit? Also, django-stubs-ext is a requirement of django-stubs, so it doesn’t need to be re-specified here; it’s enough to have the right commit in `common.in`.

Reply:
Because master is currently depending on an unpublished version of `django-stubs-ext` (`>=0.6.0`), when compiling `mypy.txt` by itself we get the following error.
```
Could not find a version that matches django-stubs-ext>=0.6.0 (from django-stubs->-r requirements/mypy.in (line 35))
Tried: 0.1.0, 0.1.0, 0.2.0, 0.2.0, 0.3.0, 0.3.0, 0.3.1, 0.3.1, 0.4.0, 0.4.0, 0.5.0, 0.5.0
There are incompatible versions in the resolved dependencies:
```

Reply:
The commit should be the same. Will get that fixed.

Reply:
Hmm, yeah. We can just remove `mypy.txt`: #22880.